Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetLog.java

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
Sorry, my mistake, no need for this class. The log GET parameters is not in request body.